cli/src/main/java/com/box/l10n/mojito/cli/command/GithubPRInfoCommand.java

```diff
@@ -63,9 +63,27 @@ public class GithubPRInfoCommand extends Command {
       description = "The Github repository owner")
   String owner;
 
+  @Parameter(
+      names = {"--skip-i18n-push-label"},
+      arity = 1,
+      required = false,
+      description = "Github label name that is used to trigger skipping i18n push")
+  String skipI18NPushLabel = "skip-i18n-push";
```

Comment:
Do we plan to customize this?

Reply:
I felt having an option to configure the label name to monitor for would be more useful than hard-coding it but for our purposes I still intend on using `skip-i18n-push`